server/routes/searchPatients/restrictedPatientSearchFilter.ts

```diff
@@ -1,41 +1,22 @@
 import { PrisonerSearchSummary } from '../../services/prisonerSearchService'
 
 export default class RestrictedPatientSearchFilter {
-  includePrisonerToMove = (prisoner: PrisonerSearchSummary): boolean => {
-    if (this.determinateSentenceAfterCRD(prisoner)) {
-      return false
-    }
-    if (this.recallAfterSED(prisoner)) {
```

Comment:
intellij suggested simplifying this `if` so followed the simplification to its conclusion 😀